### DISCUSSION TITLE
Vercel support for Jetbrains Space

### DISCUSSION THREAD

**Discussion by cryptiklemur** [vercel/community#960] [Idea]
https://www.jetbrains.com/space/

Would love to see Space get Vercel <-> Git integration!

**Comment by amyegan** [comment]
Thanks for the suggestion!

**Comment by almaron** [comment]
Yes. Absolutely yes. Have been moving my team there with all the projects and it's just such a shame it's not supported. Currently working around it by mirroring Git repos but that's a pain and not a workable solution for the long run.

**Comment by gillescoolen** [comment]
Fully agree, would be nice to see support for Space!

**Comment by laggu** [comment]
is there any progressing??
look forward to working with Jetbrains Space

**Reply by versecafe** [reply]
for a temporary fix you can always use spaces sync with github and have Vercel run off the github repo
